src/transformers/models/wav2vec2/tokenization_wav2vec2.py

```diff
@@ -277,6 +277,74 @@ def save_vocabulary(self, save_directory: str, filename_prefix: Optional[str] =
 
         return (vocab_file,)
 
+    def _add_tokens(self, new_tokens: Union[List[str], List[AddedToken]], special_tokens: bool = False) -> int:
+        # copy past the function from `src/transformers/tokenization_utils.py`
```

Comment:
If it would be an exact copy paste of the add token function I guess we wouldn't need it here. But it seems that it's not an exact copy so that we should delete the comment IMO

Reply:
Actually I took the following tempelate from issue #10622 provided by you.
So those comments were left as it is for the reference. I will remove those.
```python
def _add_tokens(self, new_tokens: Union[List[str], List[AddedToken]], special_tokens: bool = False) -> int:
   # copy past the function from `src/transformers/tokenization_utils.py` 
   # + add the "hack": 
   unique_no_split_tokens = tokenizer.unique_no_split_tokens
   tokenizer.unique_no_split_tokens = unique_no_split_tokens
```